src/Model/ModelDataCustomised.php

```diff
@@ -24,10 +24,10 @@ public function __construct(ModelData $originalObject, ModelData $customisedObje
     public function __call($method, $arguments)
     {
         if ($this->customised->hasMethod($method)) {
-            return call_user_func_array([$this->customised, $method], $arguments ?? []);
+            return $this->customised->$method(...$arguments);
         }
 
-        return call_user_func_array([$this->original, $method], $arguments ?? []);
+        return $this->original->$method(...$arguments);
```

Comment:
Needed because `call_user_func_array` shows up in the stack trace and would result in throwing the exception instead of ignoring it. See https://github.com/silverstripe/silverstripe-framework/actions/runs/12062241360/job/33635620688?pr=11482